packages/TelegramClient-Tests.package/TCTUsersTests.class/instance/userFirstName.st

```diff
@@ -0,0 +1,4 @@
+accessing
+userFirstName
+
+	^ 'Max'
```

Comment:
Is it possible to just create a MockUser out of those user properties (namely `userFirstName`, `userId`, `userLastName`, `username`)? Would be much better I think. And this could then be moved into TCTMocks as the other things @rsommerfeld suggested.

Reply:
all mocks are created in `TCTMocks` now